doc/classes/Array.xml

```diff
@@ -6,20 +6,35 @@
 	<description>
 		Generic array which can contain several elements of any type, accessible by a numerical index starting at 0. Negative indices can be used to count from the back, like in Python (-1 is the last element, -2 the second to last, etc.).
 		[b]Example:[/b]
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var array = ["One", 2, 3, "Four"]
 		print(array[0]) # One.
 		print(array[2]) # 3.
 		print(array[-1]) # Four.
 		array[2] = "Three"
 		print(array[-2]) # Three.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+		var array = new Godot.Collections.Array{"One", 2, 3, "Four"}
+		GD.Print(array[0]); // One.
+		GD.Print(array[2]); // 3.
+		GD.Print(array[-1]); // Four.
```

Comment:
Currently, negative indices on a `Godot.Collections.Array` crash the game with the error `System.IndexOutOfRangeException: Index was outside the bounds of the array`. We can add support for this (by wrapping with `Mathf.PosMod` in `Array`'s array index operator), but it would come at some performance cost, which might not be ideal considering many people choose C# for performance reasons. What do you think @neikeq ?

Reply:
My fault. I wanted to add support for this in my tool using array.Count - 1 and oversaw that. fixed

Reply:
Added some sort of support into my tool.